pages/test_analytics/javascript_collectors.md

```diff
@@ -121,6 +122,30 @@ To configure Mocha:
     }
     ```
 
+### Cypress
+To configure Cypress:
+
+1. Make sure Cypress runs with access to [CI environment variables](/docs/test-analytics/ci-environments).
+1. Update your [Cypress Configuration](https://docs.cypress.io/guides/references/configuration)
```

Comment:
Thanks for your contribution @nprizal 😄 

Just a minor suggestion, 'Configuration' is not a proper noun so I'd make this a small 'c'. Also in line with the previous step, I've added a full stop/period after the end of the line.

```suggestion
1. Update your [Cypress configuration](https://docs.cypress.io/guides/references/configuration).
```

Also, I've been updating our documentation style guide, such that we now have separate writing and Markdown syntax style guides. (The latter guide still has content about how the docs repo is structured, but that'll be brought out into its own separate 'contributors guide' at somepoint soon.)

Anyway, here's a bit about [usage of capitalisation - particularly with Product Names](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#product-names-and-product-features).